docs/source/examples/ddd_tutorial.myst

```diff
@@ -0,0 +1,261 @@
+def get_circuit(depth: int, seed: int) -> Tuple[cirq.Circuit, str]:
+    circuit, correct_bitstring = benchmarks.generate_mirror_circuit(
+        nlayers=depth,
+        two_qubit_gate_prob=1.0,
+        connectivity_graph=topology,
+        two_qubit_gate_name="CNOT",
+        seed=seed,
+        return_type="cirq",
+    )
+    return circuit, correct_bitstring
+```
+
+## Define the executor
+
+Now that we have a circuit, we define the `execute` function which inputs a circuit and returns an expectation value - here, the
+frequency of sampling the correct bitstring.
+
+```{code-cell} ipython3
+def execute(
+    circuits: cirq.Circuit,
+    shots: int,
+    correct_bitstring: str,
+    is_noisy: bool = True,
+) -> List[float]:
+    """Executes the input circuit(s) and returns ⟨A⟩, where
+    A = |correct_bitstring⟩⟨correct_bitstring| for each circuit.
+    """
+    if not isinstance(circuits, list):
+        circuits = [circuits]
+
+    # Store all circuits to run in list to be returned.
+    to_run = []
+
+    # Compile circuits to appropriate gateset.
+    for circuit in circuits:
+        circuit_to_run = circuit.copy()
+        if is_noisy:
+            to_run.append(circuit_to_run.with_noise(cirq.amplitude_damp(0.05)))
+        else:
+            to_run.append(circuit_to_run)
+
+    backend = cirq.Simulator()
+
+    # Process each job.
+    results: List[float] = []
+    for circ_to_run in to_run:
+        counts = 0
+        for _ in range(shots):
+            result = backend.run(
+                circ_to_run + cirq.measure(*sorted(circuit.all_qubits()))
+            )
+            bitstring = list(result.measurements.values())[0][0].tolist()
+            if bitstring == correct_bitstring:
+                counts += 1
+        results.append(counts / shots)
+
+    return results
+```
+
+## Select the DDD sequences to be applied
+We now import a DDD _rule_ from Mitiq, i. e., a function that generates DDD sequences of different length.
+In this example, with simulated amplitude damping noise, we opt for YY sequences (pairs of Pauli Y operations).
+
+```{code-cell} ipython3
+from mitiq import ddd
+
+rule = ddd.rules.yy
+```
+
+## Sample bitstrings from mirror circuits
+
+```{code-cell} ipython3
+:tags: [skip-execution]
+
+true_values, noisy_values = [], []
+ddd_values = []
+
+noise_scaled_expectation_values = []
+
+for depth in depths:
+    print("Status: On depth", depth, end="\n\n")
+
+    true_depth_values, noisy_depth_values = [], []
+    ddd_depth_values = []
+
+    noise_scaled_expectation_depth_values = []
+
+    for trial in range(trials):
+        # Local seed is calculated in this way to ensure that we don't get repeat values in loop.
+        local_seed = 10**6 * depth + 10**3 * seed + trial
+
+        circuit, correct_bitstring = get_circuit(depth, local_seed)
+
+        (true_value,) = execute(circuit, shots, correct_bitstring, is_noisy=False)
+        (noisy_value,) = execute(circuit, shots, correct_bitstring, is_noisy=True)
```

Comment:
```suggestion
        true_value = execute(circuit, shots, correct_bitstring, is_noisy=False)
        noisy_value = execute(circuit, shots, correct_bitstring, is_noisy=True)
```
This should work once the executor returns a single float instead of a list.

Reply:
Nice, this works!